doc/HLD-Data-Integrity.md

```diff
@@ -0,0 +1,145 @@
+# HLD of Data Integrity
+
+- I. Motr Client
+  - I.1 Application and motr data structure
+  - I.2 Parity Group Computation 
+  - I.3 Tracking Data Unit Allocated to Object
+  - I.4 FOP Sending and Checksum Processing
+    - Write Path
+    - Read Path
+- II. Motr Server Write Path
+  - II.1 GOB => COB
```

Comment:
I would recommend to use the full form here

Reply:
done